sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/ier12-integration.scheduler.e2e-spec.ts

```diff
@@ -390,4 +390,147 @@ describe(describeProcessorRootTest(QueueNames.IER12Integration), () => {
       `${assessmentId}${firstDisbursementId}${defaultApplicationNumber}1           242963189Doe                      John           19980113B   SI       Address Line 1           Address Line 2           Victoria                 BC  Z1Z1Z1          Some Program With DescripSome program with description too long            graduateDiploma          0001    5   0512123401234ADR2XYZ                      62000081620001010000033330000004444000000555500000066660019100F2000060120002001COMP20000601000010000000006000000004800000N NNY            20000602        000321200000000160000003212000000000000000000000001NNN000000000000000000000000000000000000000000000000000000000000NN0000000000000144430000000000000000000000000000000000007777000000070045000000000000000000000000000000000000000000000000000000002200000065004000046000000005500000 0000000000000000000000000000000000000000000000000000000000000000000000DISR20000811        Completed Pending   20000816                        CSLF0000100000BCSL0000600000CSGP0000200000CSGD0000300000CSGF0000400000CSGT0000500000BCAG0000700000SBSD0000900000BGPD0000800000    0000000000`,
     );
   });
+
+  it("Should generate an IER12 file with one record for a student with one sent disbursement that had a feedback error reported (DISE).", async () => {
+    // Arrange
+    const testInputData = {
+      student: JANE_MONONYMOUS_FROM_OTHER_COUNTRY,
+      application: {
+        applicationNumber: defaultApplicationNumber,
+        studentNumber: undefined,
+        relationshipStatus: RelationshipStatus.Single,
+        applicationStatus: ApplicationStatus.Completed,
+        applicationStatusUpdatedOn: undefined,
+      },
+      assessment: {
+        triggerType: AssessmentTriggerType.OriginalAssessment,
+        assessmentDate: undefined,
+        workflowData: WORKFLOW_DATA_SINGLE_INDEPENDENT_WITH_NO_DEPENDENTS,
+        assessmentData: ASSESSMENT_DATA_SINGLE_DEPENDENT,
+        disbursementSchedules: [
+          {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+            disbursementDate: undefined,
+            updatedAt: undefined,
+            dateSent: undefined,
+            disbursementValues: AWARDS_SINGLE_DISBURSEMENT_NO_BC_FUNDING,
+          },
+        ],
+      },
+      educationProgram: PROGRAM_GRADUATE_DIPLOMA_WITH_INSTITUTION_PROGRAM_CODE,
+      offering: OFFERING_FULL_TIME,
+    };
+    const application = await saveIER12TestInputData(db, testInputData, {
+      programYearPrefix: sharedProgramYearPrefix,
+      submittedDate: referenceSubmissionDate,
+    });
+    // Create a feedback error associated with the disbursement.
+    const [errorCode] = FULL_TIME_DISBURSEMENT_FEEDBACK_ERRORS;
+    const [disbursementSchedule] =
+      application.currentAssessment.disbursementSchedules;
+    const feedbackError = createDisbursementFeedbackError(
```

Comment:
Minor. Should we follow the naming convention of createFake*?

Reply:
Changed.